pkg/collectors/devices/device_info_test.go

```diff
@@ -73,6 +73,8 @@ var _ = Describe("NewContainerContext", func() {
 			firmwareVersion := "4.20 0x8001778b 1.3346.0"
 			driverVersion := "1.11.20.7"
 
+			response["ls /sys/class/net/aFakeInterface/device/gnss/"] = []byte(gnssDev)
```

Comment:
I see this positive case, but no test for when this fails or returns empty.  Is that important?

Reply:
Good spot, I had just made the old tests work. I've added the negative test :)